moved/src/state_actor.rs

```diff
@@ -650,11 +650,17 @@ impl<
     > StateActor<S, P, H, R, G, L1G, L2G, B, Q, M, InMemoryStateQueries>
 {
     pub fn on_tx_in_memory() -> OnTx<Self> {
-        Box::new(|| Box::new(|state, changes| state.state_queries.add(changes)))
+        Box::new(|| Box::new(|_state, _changes| ()))
     }
 
     pub fn on_tx_batch_in_memory() -> OnTxBatch<Self> {
-        Box::new(|| Box::new(|state| state.state_queries.tag()))
+        Box::new(|| {
```

Comment:
nit: The curly brackets here and below seem not needed.

Reply:
I think the formatter put them there automatically because the closures span multiple lines.